database/buildSchema/07B_template.sql

```diff
@@ -8,11 +8,12 @@ CREATE TYPE public.template_status AS ENUM (
 CREATE TABLE public.template (
     id serial PRIMARY KEY,
     name varchar,
-    code varchar NOT NULL,
+    code varchar NOT NULL UNIQUE,
```

Comment:
I don't think `code` can be UNIQUE since we can have multiple versions of the same template -- the `AVAILABLE` one is the "real" one.

Reply:
* remove constraint
* added categories and filters before templates
* joins categories and filters to templates in template mutation
* changes core_actions to core_mutations and added joinFilters to it

069c0c86f7f6df89fc2741ac7537c022bcfa10e5